boards/sim/sim/sim/README.txt

```diff
@@ -1245,6 +1245,18 @@ touchscreen
   See apps/examples/README.txt for further information about build
   requirements and configuration settings.
 
+toywasm
+
+  This is a configuration with toywasm.
+
+  An example usage:
+
+    NuttShell (NSH) NuttX-10.4.0
+    nsh> mount -t hostfs -o fs=/tmp/wasm /mnt
+    nsh> toywasm --wasi /mnt/hello.wasm
+    hello
+    nsh>
+
```

Comment:
Please move to Documentation/
We are trying to phase out all boards README.txt and keep everything in the web site.

Reply:
do you mean to block this PR?